RUFAS/output_manager.py

```diff
@@ -317,6 +318,19 @@ def _exclude_info_maps(self, pool: Dict[str, pool_element_type]) -> Dict[str, po
                 value.pop("info_maps")
         return pool_copy
 
+    def _list_txt_file_names_in_dir(self, dir_path: str) -> List[str]:
+        """ Returns the list of files in the given path"""
+        dir_path_check = Path(dir_path)
+        if dir_path_check.is_dir():
+            txt_files = []
+            all_files = os.listdir(dir_path)
+            for filename in all_files:
+                if filename.endswith(".txt"):
+                    txt_files.append(filename)
+            return txt_files
+        else:
+            raise NotADirectoryError("specified path is not a directory")
```

Comment:
'NotADirectoryError' already says what's wrong, the message string can be used to guide the user on what is needed.

```suggestion
            raise NotADirectoryError("The specified path must be a directory")
```